content/quiz_events.md

```diff
@@ -95,6 +95,40 @@ Below are the the scoring calculation rules and the logic for them. This procedu
 **Team Roster**
 : This is an array of quizzer objects, each of which contains a "correct answers" value
 
+### How to Score Individual and Team Points
+**Individual Points**
+
+_**Points Earned**_
+- +20 Points for every correct question & toss-up question
+- +10 Points for every quiz out without error
+
+**_Points Deducted_**
+- -10 Points for 2nd and subsequent personal errors
+     - Points are not deducted for errors on bonus questions
+     - Points are not deducted for errors during overtime
+- -10 Points for 3rd personal foul
+
+
+**Team Points**
+All point earned or deducted by an individual are to be counted towards the team’s points.
+
+_**Points Earned**_
+- +20 Points for team arriving on time. 
+     - Teams that arrive late due to quizzing in another room are excused from the forfeiture of these points. If a single quizzer is late, the coach can decide to keep the +20 points by keeping the late quizzer out the whole quiz, or forfeit the points by subbing the quizzer in after question number 1. This decision must occur before the quiz has been started. 
+- +10 Points for every 3rd and subsequent correct answer given by a unique quizzer on the team
+- +20 Points for every correct bonus question before question number 17
+     - In 2 Team Quizzes, this scoring rule does not apply 
+- +10 Points for every correct bonus question after and including question number 17
+     - In 2 Team Quizzes, all correct bonus questions are worth 10 points
+
+_**Points Deducted**_
+- -10 Points for every team error starting at error number 3
```

Comment:
team error needs to be defined as "the sum of all individual, non-bonus errors"

probably also need to state that bonus errors are not negative points. 

probably also need to state that a team can never receive more than -10 on a single question, due to ERRORS. i.e. error negative points cannot be stacked on a single question.  

probably also need to define "single question" somewhere--as distinct to a "single numbered question"

Reply:
_**Points Deducted**_
- -10 Points for every team error starting at error number 3
     - Team errors are the sum of all individual, non-bonus errors
- Points are not deducted for an error on a bonus question.
- No more than -10 points can be deducted per question asked.

Reply:
for that last bullet, need to specify that it's specific to errors.  You CAN stack negative points (error, challenge, protest, foul)

Reply:
_**Points Deducted**_
- -10 Points for every team error starting at error number 3
     - Team errors are the sum of all individual, non-bonus errors
- Points are not deducted for an error on a bonus question.
- No more than -10 points can be deducted per question asked due to an error.

Reply:
or should I add another line saying

_**Points Deducted**_
- -10 Points for every team error starting at error number 3


_**Points Deducted**_
- -10 Points for every team error starting at team error number 3
     - Team errors are the sum of all individual, non-bonus errors
     - Points are not deducted for an error on a bonus question.
- -10 Points for every error starting at question number 17
- -10 Points for 4th and subsequent team fouls
- -10 Points for 2nd and subsequent overruled challenges & protests
- -10 Points at the determination of the room officials that a deliberate attempt was made to forfeit a question. 
- No more than -10 points can be deducted per question asked due to an error.
- Non-error related deducted points are cumulative.

Reply:
Hmmmm, I'd think about the order some. 

You use "team error" before you define it.

And for the last 2 bullets, I think the 2nd bullet should be nested under the first?

Reply:
Sorry, the first line duplicated. 

Are you saying that I should have a bulleted line defining team error, and then start the list?


_**Points Deducted**_
Team errors are the sum of all individual, non-bonus errors
- -10 Points for every team error starting at team error number 3
     - Points are not deducted for an error on a bonus question.
- -10 Points for every error starting at question number 17
- -10 Points for 4th and subsequent team fouls
- -10 Points for 2nd and subsequent overruled challenges & protests
- -10 Points at the determination of the room officials that a deliberate attempt was made to forfeit a question. 
- No more than -10 points can be deducted per question asked due to an error.
     - Non-error related deducted points are cumulative.

Reply:
Yes I think that's cleaner. 

I don't mind this reiteration "Points are not deducted for an error on a bonus question." but it IS redundant, since you defined Team Errors already, clearly stating that bonus errors aren't team errors.

Reply:
I think keeping that in clarifies the next line, unless it can be reworded. 

-10 Points for every error starting at question number 17

I don't think changing error to team error makes sense in the case

Reply:
-10 Points for every error on a question or toss-up question starting at question number 17